src/client/components/AddImageToRoom/AddImageToRoom.stories.js

```diff
@@ -0,0 +1,6 @@
+import React from 'react';
+import AddImageToRoom from './AddImageToRoom';
+
+export default { title: 'Add Image To Room' };
+
+export const Component = () => <AddImageToRoom AddRoomImage={() => {}} />;
```

Comment:
Are you planning to add prop `AddRoomImage` in the future? If not, this is extra.